ARSIS-Telemetry-GroundControl/telemetry-api/app/main.py

```diff
@@ -1,4 +1,5 @@
 from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
 from app.routers import biometrics, location, user, uia
 from app.caches.user_cache import UserCache
```

Comment:
Delete line

Reply:
I think i need that import in a later line... i might be wrong